### PR TITLE
React-ified Error Handler

### PR DESCRIPTION
**Pull request by ash3rz**
image

### CODE REVIEW

File: react-components/src/util/ErrorHandler.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            errorDetails,
+            classes,
+        } = this.props;
+
+        return (
+            <Dialog open={open}
+                    onClose={this.onClose}
+                    id={ids.errorHandlerDlg}
+            >
+                <DialogTitle>
```
Comment (sri2k1us) [inline react-components/src/util/ErrorHandler.js]: I mentioned this in Slack. Once Analyses PR is merged, we could use DEDialogHeader here to get uniform styling.

**Comment by ash3rz** [comment]
@sri2k1us Which class were you mentioning?

**Comment by sri2k1us** [comment]
> @sri2k1us Which class were you mentioning?

/de-lib/src/main/java/org/iplantc/de/commons/client/views/ReactUtil.java

**Comment by ash3rz** [comment]
Ah ok, I was thinking eventually we'd have more added to `ReactUtil` but I can rename it since for now it is the only thing using it 👍

**Comment by psarando** [comment]
> Ah ok, I was thinking eventually we'd have more added to `ReactUtil` but I can rename it since for now it is the only thing using it 👍

Actually I agree, I think it makes sense to keep it `ReactUtil` since this class is named for the namespace `CyVerseReactComponents.util`, and it won't have to be renamed if we export more classes from it in the future.

**Comment by sri2k1us** [comment]
> > Ah ok, I was thinking eventually we'd have more added to `ReactUtil` but I can rename it since for now it is the only thing using it 👍
> 
> Actually I agree, I think it makes sense to keep it `ReactUtil` since this class is named for the namespace `CyVerseReactComponents.util`, and it won't have to be renamed if we export more classes from it in the future.
@psarando - Sorry am being picky here. If thats the case, I would rename it to ReactUtilComponents. A generic name like ReactUtil only confuses people to think it is React Utility class.

**Comment by psarando** [comment]
> @psarando - Sorry am being picky here. If thats the case, I would rename it to ReactUtilComponents. A generic name like ReactUtil only confuses people to think it is React Utility class.

@sri2k1us Yeah, actually that makes sense. Something like `ReactUtilComponents` or `ReactViewUtils` would be more clear 👍

**Comment by ash3rz** [comment]
I agree, I think I'll rename to `ReactUtilComponents` and then merge.

**Comment by ash3rz** [comment]
Thanks for the reviews!

**Pull request merged**